wp-content/themes/eightshift-boilerplate/src/class-config.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * The file that defines the main start class.
+ *
+ * A class definition that includes attributes and functions used across both the
+ * theme-facing side of the site and the admin area.
+ *
+ * @package Eightshift_Boilerplate\Core
+ */
+
+declare( strict_types=1 );
+
+namespace Eightshift_Boilerplate\Core;
+
+use Eightshift_Libs\Core\Config as Lib_Config;
+
+/**
+ * The project config class.
+ *
+ * @since 2.0.0
+ */
+class Config extends Lib_Config {
+
+  /**
+   * Method returns project name generally used for naming assets handlers, languages and etc.
+   *
+   * @since 2.0.0
+   */
+  public static function get_project_name() : string {
+    return 'eightshift-boilerplate';
+  }
+
+  /**
+   * Method returns project version generally used for versioning assets handlers.
+   *
+   * @since 2.0.0
+   */
+  public static function get_project_version() : string {
+    return '1.0.0';
+  }
+
+  /**
+   * Returns project prefix for adding it to all the filters as a prefix because all filters in WordPress live inside a global namespace.
+   *
+   * @return string Full path to asset.
+   *
+   * @since 2.0.0 Init
```

Comment:
```suggestion
   * @since 2.0.0 Added in the project
```

Maybe this is better than just `Init`.

Reply:
👍